examples/seat-adjuster/src/SeatAdjusterApp.cpp

```diff
@@ -126,8 +127,17 @@ void SeatAdjusterApp::onErrorTopic(const velocitas::Status& status) {
 }
 } // namespace example
 
+std::unique_ptr<example::SeatAdjusterApp> myApp;
+
+void signal_handler(int sig) {
```

Comment:
We should think about moving this to a common place in the SDK, maybe introducing a `Launcher` which spawns the app registers the signal handler?

e.g.
```cpp
template<typename TAppClass> 
class Launcher {
public:
   Launcher() = default;
   ~Launcher() = default;

  void signalHandler() { ... }

   template<typename TArgs...>
   void launch(TArgs... args) {
      m_app = std::make_unique<TAppClass>(...args);
      signal(SIGINT, &Launcher<TAppClass>::signalHandler);
   }


private:
   std::unique_ptr<TAppClass> m_app;
}
```

Reply:
Shall we create a ticket for this?

Reply:
Issue created: #90